### PR TITLE
cargo fmt for rust 1.29

### PR DESCRIPTION
**Pull request by robertDurst**
### Is there a GIF that reflects how this work made you feel?
noice

### CODE REVIEW

**Review by correlator — APPROVED** [review]
🚀 
Looks like find path is hanging up again...I suppose we can increase the timeout again @kbacha ?

File: cli/src/trades.rs [hunk 1]
```diff
@@ -86,8 +86,8 @@ pub fn aggregations(client: &Client, matches: &ArgMatches) -> Result<()> {
             .parse::<DateTime<Utc>>()
             .map_err(|_| {
                 String::from("Start time should be in ISO 8601 format, ex: x2017-11-28T12:00:09Z")
-            })?
-            .timestamp() as u64 * 1_000;
+            })?.timestamp() as u64
+            * 1_000;
```
Comment (choubacha) [inline cli/src/trades.rs:90]: Whoa, not sure I agree with this one, `cargo-fmt`

File: client/src/endpoint/trade.rs [hunk 2]
```diff
@@ -299,8 +299,7 @@ mod all_trades_tests {
                     "MOBI",
                     "GA6HCMBLTZS5VYYBCATRBRZ3BZJMAFUDKYYF6AH6MVCMGWMRDNSWJPIH",
                 ),
-            )
-            .with_offer_id(123)
+            ).with_offer_id(123)
```
Comment (choubacha) [inline client/src/endpoint/trade.rs:302]: Or this one 🤔

File: client/src/resources/transaction.rs [hunk 3]
```diff
@@ -7,7 +7,11 @@ use chrono::prelude::*;
 /// To learn more about the concept of memos in the Stellar network, take a look at the Stellar memo description here:
 /// <https://www.stellar.org/developers/guides/concepts/transactions.html#memo>
 #[derive(Deserialize, Debug, Clone, PartialEq, Eq)]
-#[serde(rename_all = "lowercase", tag = "memo_type", content = "memo")]
+#[serde(
+    rename_all = "lowercase",
+    tag = "memo_type",
+    content = "memo"
```
Comment (choubacha) [inline client/src/resources/transaction.rs:13]: I like this one though!

**Review by choubacha — APPROVED** [review]

**Comment by choubacha** [comment]
Silly find path always breaking on timeouts.

**Pull request merged**